weblate/checks/format.py

```diff
@@ -439,6 +462,17 @@ class LuaFormatCheck(BasePrintfCheck):
     description = _("Lua format string does not match source")
 
 
+class SchemeFormatCheck(BasePrintfCheck):
+    """Check for Scheme format string."""
+
+    check_id = "scheme_format"
+    name = _("Scheme format")
+    description = _("Scheme format string does not match source")
+
+    def normalize(self, matches):
+        return [m for m in matches if m != "~"]
+
+
```

Comment:
This will be most likely needed to properly display hints on missing format:

```suggestion

    def format_string(self, string):
        return f"~{string}"

```

Reply:
oh, so that's what it does :D, thanks!